pkg/storegateway/indexcache/remote.go

```diff
@@ -32,10 +32,10 @@ const (
 )
 
 var (
-	postingsCacheKeyLabelHashBufferPool = zeropool.New[[]byte](func() []byte {
-		// We assume the label name/value pair is typically not longer than 1KB.
```

Comment:
Didn't you like my comment? :D

Reply:
Oh, I initially changed to return `new([]byte)` and allocate a `max(expectedLen, 1024)` in the usage, but that seemed unnecessary complex to me, so I lost the comment when I switched back. Recovered it now :)